src/panel/PanelManager.jsx

```diff
@@ -211,13 +217,35 @@ export default class PanelManager extends React.Component {
     this.setState({ panelSize });
   }
 
+  closeBetaPopup = () => {
+    setBetaPopupClosed();
+    this.setState({ betaPopupClosed: true });
+  }
+
   render() {
-    const { ActivePanel, options, panelSize } = this.state;
+    const { ActivePanel, options, panelSize, betaPopupClosed, client } = this.state;
 
     return <PanelContext.Provider value={{ size: panelSize, setSize: this.setPanelSize }} >
       <div className="panel_container">
         <ActivePanel {...options} />
       </div>
+
+      {
+        // Show beta popup if browser language is not french, if popup hasn't already been closed, and if the user comes from an IA
+        !navigator.language.includes('fr')
```

Comment:
As the condition and state management is complex enough, why not encapsulating all this in a dedicated component? I suggest a name like `<BetaDisclaimer>`.
You could move everything related to it in it and render `null` when it's not needed. That way you could just render it without condition and wouldn't need to change anything in the props and state of `PanelManager`, as it's already tricky.
BTW, as this disclaimer isn't related to the panels at all, it would make even more sense to put it in `RootComponent` instead of `PanelManager`.